diff -up tempfile-3.2.0/Cargo.toml.1~ tempfile-3.2.0/Cargo.toml
--- tempfile-3.2.0/Cargo.toml.1~	2021-03-25 00:55:36.914642253 +0100
+++ tempfile-3.2.0/Cargo.toml	2021-03-25 00:56:12.141873503 +0100
@@ -30,10 +30,5 @@ version = "0.8"
 
 [dependencies.remove_dir_all]
 version = "0.5"
-[target."cfg(target_os = \"redox\")".dependencies.redox_syscall]
-version = "0.2"
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.27"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = ["fileapi", "handleapi", "winbase"]
